src/app/services/client.ts

```diff
@@ -1,6 +1,24 @@
-import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { ApolloClient, createHttpLink, InMemoryCache, NormalizedCacheObject } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+import { AUTH_TOKEN_KEY } from 'app/constants';
 
-export const Client = new ApolloClient({
-  uri: 'https://tq-template-server-sample.herokuapp.com/graphql',
-  cache: new InMemoryCache(),
-});
+export function client(): ApolloClient<NormalizedCacheObject> {
```

Comment:
acho que tem que mudar o nome dessa função, somente `client` não tem muito significado

Reply:
Será que vale a pena isso ser função mesmo? Cada vez que ela for executada ela vai criar um client diferente. Precisam ser diferentes os clients msm?